apps/gradient_autoscheduler/GradientAutoscheduler.cpp

```diff
@@ -154,6 +154,59 @@ void parallelize_vars_and_rvars_gpu(
         serial_rvars = rvars;
     }
 
+    if (gpu_threads.empty() && r_gpu_threads.empty()) {
+        // If we can't find any loop large enough for a GPU thread,
```

Comment:
Nit: this comment seems paradoxical: if none of the loops are large enough for a gpu thread, then presumably even the largest remaining is still not large enough. Clarify?

Reply:
Fixed!